pyrsh.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+import paramiko
+import argparse
+
+def Main():
```

Comment:
@GitSergeyGit lets follow python style gid for function/method names, this means it should be lowercase